atlassian/bitbucket/cloud/repositories/users.py

```diff
@@ -0,0 +1,75 @@
+from requests.models import ProtocolError
+from ..base import BitbucketCloudBase
+from datetime import datetime
+
+
+class User(BitbucketCloudBase):
+    def __init__(self, url, data, *args, **kwargs):
+        super(User, self).__init__(url, *args, data=data, expected_type="user", **kwargs)
+
+    @property
+    def display_name(self):
+        return str(self.get_data("display_name"))
+
+    @property
+    def nickname(self):
+        return self.get_data("nickname")
+
+    @property
+    def account_id(self):
+        return self.get_data("account_id")
+
+    @property
+    def uuid(self):
+        return self.get_data("uuid")
+
+    def delete(self):
+        """
+        Deletes the default reviewer
+        """
+        data = super(User, self).delete(None)
+        if "errors" in data:
+            return
+        return User(self.url, data, **self._new_session_args)
+
+
+class Participant(BitbucketCloudBase):
```

Comment:
should we add a `is_default_reviewer` property?

Reply:
For the Participiant make sense.

Reply:
This should be implemented in the next iteration. #662